boards/wiznet/w5500_evb_pico2/doc/index.rst

```diff
@@ -0,0 +1,146 @@
+.. zephyr:board:: w5500_evb_pico2
+
+Wiznet W5500 Evaluation Board Pico2
+###################################
+
+Overview
+********
+
+W5500-EVB-Pico2 is a microcontroller evaluation board based on the Raspberry
+Pi RP2350A and fully hardwired TCP/IP controller W5500 - and basically works
+the same as Raspberry Pi Pico2 board but with additional Ethernet via W5500.
+The USB bootloader allows the ability to flash without any adapter, in a
+drag-and-drop manner. It is also possible to flash and debug the boards with
+their SWD interface, using an external adapter.
+
+Hardware
+********
+
+- Dual core Arm Cortex-M33 or Hazard3 processor running up to 133MHz
+- 520KB on-chip SRAM
+- 16MB on-board QSPI flash with XIP capabilities
+- 26 GPIO pins
+- 3 Analog inputs
+- 2 UART peripherals
+- 2 SPI controllers
+- 2 I2C controllers
+- 16 PWM channels
+- USB 1.1 controller (host/device)
+- 3 Programmable I/O (PIO) for custom peripherals
+- On-board LED
+- 1 Watchdog timer peripheral
+- Wiznet W5500 Ethernet MAC/PHY
+
+
+.. figure:: img/w5500_evb_pico2.webp
+     :align: center
+     :alt: W5500 Evaluation Board
+
+     Wiznet W5500_EVB_PICO evaluation board (Image courtesy of Wiznet)
```

Comment:
Zephyr board sphinx directive takes care of adding the image to the overview box 

```suggestion
```